src/practices/JavaScript/ESlintWithoutErrorsPractice.spec.ts

```diff
@@ -24,32 +25,289 @@ describe('ESLintWithoutErrorsPractice', () => {
+              nodeType: 'Literal',
+              messageId: 'wrongQuotes',
+              endLine: 1,
+              endColumn: 58,
+              fix: [Object],
+            },
+          ],
+          errorCount: 1,
+          warningCount: 0,
+          fixableErrorCount: 1,
+          fixableWarningCount: 0,
+          source: '',
+        },
+      ],
+    };
     mockedEslint.mockImplementation(() => {
       return {
         executeOnFiles: () => report,
       };
     });
     const result = await practice.evaluate(containerCtx.practiceContext);
+    expect(practice.data.statistics?.linterIssues).toBeDefined();
+    expect(practice.data.statistics?.linterIssues![0]).toEqual({
+      filePath: '/Users/jakubvacek/dx-scanner/src/commands/init.ts(1)(37)',
+      severity: 'error',
+      type: 'Strings must use doublequote.',
+      url: '/Users/jakubvacek/dx-scanner/src/commands/init.ts',
+    });
+    expect(result).toEqual(PracticeEvaluationResult.notPracticing);
+  });
+
+  it('Returns not practicing, if errorCount !== 0, creates correct linter issue dtos if there are errors, gitHub repository', async () => {
+    const report = {
+      errorCount: 1,
+      results: [
+        {
+          filePath: '/var/folders/65/4xfpkxbj6k14928d02s33cqm0000gn/T/dx-scannercVQa58/src/commands/init.ts',
+          messages: [
+            {
+              ruleId: 'quotes',
+              severity: 2,
+              message: 'Strings must use doublequote.',
+              line: 1,
+              column: 37,
+              nodeType: 'Literal',
+              messageId: 'wrongQuotes',
+              endLine: 1,
+              endColumn: 58,
+              fix: [Object],
+            },
+          ],
+          errorCount: 1,
+          warningCount: 0,
+          fixableErrorCount: 1,
+          fixableWarningCount: 0,
+          source: '',
+        },
+      ],
+    };
+    mockedEslint.mockImplementation(() => {
+      return {
+        executeOnFiles: () => report,
+      };
+    });
+    containerCtx.practiceContext.projectComponent.path = '/var/folders/65/4xfpkxbj6k14928d02s33cqm0000gn/T/dx-scannercVQa58';
+    containerCtx.practiceContext.projectComponent.repositoryPath = 'https://github.com/DXHeroes/dx-scanner';
+    const result = await practice.evaluate(containerCtx.practiceContext);
+    expect(practice.data.statistics?.linterIssues).toBeDefined();
+    expect(practice.data.statistics?.linterIssues![0]).toEqual({
+      filePath: '/var/folders/65/4xfpkxbj6k14928d02s33cqm0000gn/T/dx-scannercVQa58/src/commands/init.ts(1)(37)',
+      severity: 'error',
+      type: 'Strings must use doublequote.',
+      url: 'https://github.com/DXHeroes/dx-scanner/tree/master/src/commands/init.ts',
+    });
+    expect(result).toEqual(PracticeEvaluationResult.notPracticing);
+  });
+
+  it('Returns not practicing, if errorCount !== 0, creates correct linter issue dtos if there are errors, gitLab repository', async () => {
+    const report = {
```

Comment:
Move this mock to some mock file. Create method such as `const mockEslintReport = (report?: Partial:<EslintReport>): EslintReport => {}`
See e.g. this file `/Users/adelka/dx-scanner/src/services/git/__MOCKS__/bitbucketServiceMockFolder/getIssueResponse.ts`